lib/lens.rb

```diff
@@ -4,8 +4,8 @@ module Lens
   HEADERS = {
     'Content-type' => 'application/json',
     'Content-Encoding' => 'deflate',
-    'Accept'       => 'text/json, application/json',
-    'User-Agent'   => "LENS-Ruby client #{VERSION}; #{RUBY_VERSION}; #{RUBY_PLATFORM}"
+    'Accept' => 'text/json, application/json',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.